spec/rails_helper.rb

```diff
@@ -8,6 +8,9 @@
 require 'steps/user_steps'
 require 'steps/approval_steps'
 
+# mimic production env for view+mail
+ENV["DISABLE_SANDBOX_WARNING"] = 'true'
```

Comment:
I think you're right. The test failures we currently have are expecting it to *not* be set.